examples/hydrogen-vqe.yaml

```diff
@@ -9,32 +9,32 @@ imports:
   type: git
   parameters:
     repository: "git@github.com:zapatacomputing/z-quantum-core.git"
-    branch: "master"
+    branch: "dev"
 - name: z-quantum-vqe
   type: git
   parameters:
     repository: "git@github.com:zapatacomputing/z-quantum-vqe.git"
-    branch: "master"
+    branch: "dev"
 - name: z-quantum-optimizers
   type: git
   parameters:
     repository: "git@github.com:zapatacomputing/z-quantum-optimizers.git"
-    branch: "master"
+    branch: "dev"
 - name: diatomic-molecule
   type: git
   parameters:
     repository: "git@github.com:zapatacomputing/tutorial-1-diatomic-molecule.git"
-    branch: "master"
+    branch: "dev"
 - name: qe-psi4
   type: git
   parameters:
     repository: "git@github.com:zapatacomputing/qe-psi4.git"
-    branch: "master"
+    branch: "dev"
 - name: qe-forest
   type: git
   parameters:
     repository: "git@github.com:zapatacomputing/qe-forest.git"
-    branch: "master"
+    branch: "dev"
```

Comment:
These should stay pointed to `master`/`main`

Reply:
Fixed in https://github.com/zapatacomputing/z-quantum-vqe/pull/25/commits/0b19397b7aff5e2bab91c24ff8b78d07cbc13801, thanks for catching!